components/gc-intro/gc-intro-block-en.html

```diff
@@ -0,0 +1,88 @@
+---
+title: Introduction block pattern
+description: Documentation and working example for the introduction block pattern
+language: en
+altLangPage: gc-intro-block-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+	<p>The introduction block pattern introduces the content of a landing page.</p>
+
+	<h2>On this page</h2>
+	<ul>
+		<!-- <li><a href="#inc-components">Included components</a></li> -->
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+
+	<h2 id="before-start">Before you start</h2>
+	<p>The introduction block pattern is a mandatory element for institutional landing pages and topic pages. It’s optional for other landing pages.</p>
+
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
+		<li><a href="">Introduction block pattern</a></li>
+	</ul>
+
+	<h2 id="accessibility-guidance">Accessibility guidance</h2>
+	<p>List of accessibility guidances to be followed to ensure component is used in respect of the accessibility guideline and EN 301 549 standard (including WCAG 2.1 Level AA).</p>
+	<p>There is <span>no accessibility guidance</span> defined.</p>
+
+	<h2 id="variations">Variations</h2>
+	<p>The introduction block pattern consists of:</p>
+	<ul>
+		<li>title</li>
+		<li>short description</li>
+	</ul>
+	<p>It can also include:</p>
+	<ul>
+		<li>image</li>
+		<li>supertask button</li>
+	</ul>
+
+	<h3>Introduction block</h3>
+	<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
+		<div class="container">
+			<div class="row mrgn-bttm-md">
+				<div class="col-md-7">
```

Comment:
What is the purpose of the CSS grid here?

Is it only to squash the content? If so what do you think if we simply apply the [content width limited](https://wet-boew.github.io/GCWeb/templates/content-page/content-limit-en.html) style on this page? 

Regarding the underline, can we assume it will be replaced by the gc-thickline once it become the new default?

Reply:
The column grid can be changed if needed, depending how people want to display the content. The image is optional.

Reply:
For your other questions:

H1: Let’s come back to it when we know how those other components will work

Info/suggestive: Right now, it has to be followed for the ILP as is in the guidance, but an intro block would be open to customize entirely for other page types until otherwise specified 

Stable component: We think it’s too early to know how we want to address it, its mandatory for the ILP right now, but wouldn’t need to be monitored elsewhere in our opinion